security/aws/rds_publicly_accessible/AWS_Publicly_Accessible_RDS_Instances.pt

```diff
@@ -0,0 +1,515 @@
+name "AWS Publicly Accessible RDS Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report and remediate any Relational Database Service (RDS) instances that are publicly accessible. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/security/aws/rds_publicly_accessible) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+category "Security"
+severity "high"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# User inputs
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_exclude_tags" do
+  type "list"
+  label "Tags to ignore"
+  description "List of tags that will exclude resources from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag keys or key/value pairs can be listed. Example: 'test,env=dev'"
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+auth "auth_us_east_1", type: "aws" do
```

Comment:
take a look at https://github.com/rightscale/policy_templates/pull/234#discussion_r284825518. You can just use one auth block.

Reply:
Looked at #234 (comment) and taken care of them. 
Thanks for suggestion, used one auth block and tested it, working fine